homeassistant/components/camera/raspberry_camera.py

```diff
@@ -0,0 +1,86 @@
+"""Camera platform that has a Raspberry Pi camera."""
```

Comment:
Can you rename this file to `rpi_camera.py` so it is in line with our other Raspberry component